pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -257,14 +255,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
+		Interval:      interval,
+		Timeout:       timeout,
+		CleanupPolicy: updateCleanupPolicy,
```

Comment:
It is not obvious to the user that using a percentage update will also change the availabilityCheck.
How about adding a command line flag for availabilityCheck. It can be "created" by default, but user should be able to change it to "ready".

This is similar to the generator flag in kubectl expose, which I like: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/expose.go#L59

(If we want to go all the way, then we can name them "created/v1" so that we can make changes to the created availability check freely)